src/actions/player/playerTrendsActions.js

```diff
@@ -51,12 +51,19 @@ export const getPlayerTrends = (playerId, options = {}, fieldName) => (dispatch)
         return cumulativeList;
       }, []);
 
-      return cumulativeSums.map((value, index) => ({
-        x: index + 1,
-        value: Number((value / (index + 1)).toFixed(2)),
-        match_id: matches[index].match_id,
-        hero_id: matches[index].hero_id,
-      }));
+      return cumulativeSums.map((value, index) => {
+        const match = matches[index];
+        return {
```

Comment:
How does this look? Or should I just put it on another line ![image](https://cloud.githubusercontent.com/assets/1994028/23124566/9ae18880-f722-11e6-986e-fc833ccce5e9.png)